internal/plugins/info.csv

```diff
@@ -0,0 +1,248 @@
+name                    ,type      ,commercial_name         ,support    ,deprecated ,cloud ,cloud_ai
```

Comment:
What about `zmq4`?

Reply:
Fdfrhaskldjghyjklsdahfgjkedhfkjladfhjv, yep. CI caught it which is good.